code/go/0chain.net/chaincore/chain/config.go

```diff
@@ -562,71 +564,71 @@ func (c *ConfigImpl) Update(fields map[string]string, version int64) error {
 	logging.Logger.Debug("Updating config", zap.Int64("old version", old), zap.Int64("new version", conf.version))
 
 	var err error
-	conf.IsStateEnabled, err = cf.GetBool(minersc.State)
+	conf.IsStateEnabled, err = cf.GetBool(enums.State)
```

Comment:
Let's keep the settings in minersc package as they are highly cohesive, we should avoid putting everything together in the same place. We can move some of them to global config package later but not the `enums` package, the name of `enums` is just not good, we will rename it later.